Configurations/unix-Makefile.tmpl

```diff
@@ -1641,12 +1641,44 @@ EOF
           my $dofile = abs2rel(rel2abs(catfile($config{sourcedir},
                                                "util", "dofile.pl")),
                                rel2abs($config{builddir}));
-          my @perlmodules = ( 'configdata.pm',
-                              grep { $_ =~ m|\.pm$| } @{$args{deps}} );
-          my %perlmoduleincs = map { '"-I'.dirname($_).'"' => 1 } @perlmodules;
-          $deps = join(' ', $deps, compute_platform_depends(@perlmodules));
-          @perlmodules = map { "-M".basename($_, '.pm') } @perlmodules;
-          my $perlmodules = join(' ', '', sort keys %perlmoduleincs, @perlmodules);
+          my @perlmodules = ();
```

Comment:
Did you consider making this a helper function?  It looks like duplicated code (but might not be).

Reply:
I did consider it...  or rather, to move some parts of the `generatesrc` functions to more common part, so the build file templates perl nugets could be made much smaller.  But, that's refactoring for another day...

Reply:
:+1: